mm2src/kdf_walletconnect/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! # WalletConnect WebSocket Client
```

Comment:
If this is a websocket specific crate, maybe we should rename `kdf_walletconnect` to something more appropriate.

Reply:
not really..I improve the comment